.github/workflows/pr-checks.yml

```diff
@@ -5,9 +5,9 @@ name: PR-checks
 
 on:
   push:
-    branches: ["develop", "feature/VTMDEV-1"]
+    branches: ["develop", "feature/VTMDEV-1", "feature/CB2-9571"]
```

Comment:
Have approved this, but may want to put a ticket in the backlog to undo this change once the holding branch has been merged into develop.